examples/erc20/DAI/debt-dai.xml

```diff
@@ -0,0 +1,327 @@
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="borrowRate">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:label>
+            <ts:string xml:lang="en">Borrowing Rate Mode</ts:string>
+        </ts:label>
+        <ts:origins>
+            <!-- strange enough, they seem to be using e23 here and we don't have as="e23" -->
+            <ethereum:call function="getBorrowRate" as="uint">
+                <ts:data>
+                    <ts:address>0x6B175474E89094C44Da98b954EedeAC495271d0F</ts:address>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="liquidityRate">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:origins>
+            <!-- strange enough, they seem to be using e23 here and we don't have as="e23" -->
+            <ethereum:call function="getLiquidityRate" as="uint">
+                <ts:data>
+                    <ts:address>0x6B175474E89094C44Da98b954EedeAC495271d0F</ts:address>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="totalCollateralETH">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:origins>
+            <ethereum:call function="getTotalCollateralETH" as="e18">
+                <ts:data>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="totalBorrowsETH">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:origins>
+            <ethereum:call function="getTotalCollateralETH" as="e18">
+                <ts:data>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="availableBorrowsETH">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:origins>
+            <ethereum:call function="getAvailableBorrowsETH" as="e18">
+                <ts:data>
+                    <ts:address ref="ownerAddress"/>
+                </ts:data>
+            </ethereum:call>
+        </ts:origins>
+    </ts:attribute>
+
+    <ts:attribute name="ltv">
+        <ts:type>
+            <ts:syntax>1.3.6.1.4.1.1466.115.121.1.36</ts:syntax>
+        </ts:type>
+        <ts:origins>
+            <!-- strange enough, they seem to be using e23 here and we don't have as="e23" -->
+            <ethereum:call function="getLtv" as="uint">
```

Comment:
@colourful-land @James-Sangalli This gets a value in the order of ~75. How should it be displayed?

The sample code to do this might not be correct:

> const ltv = result[6].div(1e25, 10).toPrecision(4);
> ${ltv} %

Reply:
James: "75%"